web-setup.sh

```diff
@@ -0,0 +1,118 @@
+#!/bin/sh
+
+# Author : Anthony Bartman
+# Description : This will install all neccessary files for
+# our Flask backend server. Mostly from this tutorial: 
+# https://www.codementor.io/@dongido/how-to-build-restful-apis-with-python-and-flask-12qto530jd
+
+echo ""
+echo ""
+echo "Running Flask Backend Install script."
+echo "This may take a few moments..."
+echo ""
+
+#Use Python 3.7 on board, not using Python2
+sudo update-alternatives --install /usr/bin/python python /usr/bin/python3 7
```

Comment:
Yup this is how it is suppose to be. I'm just changing the default from Python2.7 to Python 3.7.3. aka for commands, using just 'python' instead of 'python3'.

Reply:
I'm changing the default setup because we are going to be deleting python2 form the board to reduce space eventually